### PR TITLE
Issue 476: Update CNSWP Readme

### PR DESCRIPTION
**Pull request by TheFoxAtWork**
This is to provide update instructions, guidance and context around sustaining and improving the CNSWP

@lumjjb lets hope i havent ruined it again.

This update completes the items in issue #476 to provide more clarity on the maintenance of the whitepaper.

Readme needs to include

 plans for updating,
 decisions behind design of the whitepaper,
 guidance on content updates (i.e. new paper versus paper update, no products listed)

### CODE REVIEW

File: security-whitepaper/README.md [hunk 1]
```diff
@@ -1,14 +1,80 @@
...
+than 3 printable pages.  The executive summary should be a brief, high level
+overview of the paper's contents for those without the immediate time to read
+the length of the document.
 
 Links:
-* [Managed version in Markdown](https://github.com/cncf/sig-security/blob/master/security-whitepaper/cloud-native-security-whitepaper.md)
-* [Posted PDF first version](https://github.com/cncf/sig-security/blob/master/security-whitepaper/CNCF_cloud-native-security-whitepaper-Nov2020.pdf)
-* [CNCF Blog Post](https://www.cncf.io/blog/2020/11/18/announcing-the-cloud-native-security-white-paper/)
-* [Kubernetes Blog Post](https://kubernetes.io/blog/2020/11/18/cloud-native-security-for-your-clusters/)
+* [Managed version in  Markdown](https://github.com/cncf/sig-security/blob/master/security-whitepaper/cloud-native-security-whitepaper.md)
```
Comment (pragashj) [inline security-whitepaper/README.md:76]: Should we specify how the status of the md file should be managed when starting to make changes there -- for example an easy way to know that it is a draft that hasn't been published as pdf yet. If so where is a good place to mention that? May be spell that out in the `versioning and publishing` section?

**Review by pragashj — APPROVED** [review]
One minor comment for update on versioning section, otherwise LGTM

**Comment by TheFoxAtWork** [comment]
@pragashj added in more clarity.  working on updating the md to specify version and the metadata table.

**Comment by TheFoxAtWork** [comment]
@pragashj all fixed.  please review
@PushkarJ please also take a look

**Comment by PushkarJ** [comment]
One suggestion, we could put a placeholder section for retrospective / Feedback loop process and link #480 in readme, mentioning that this is work in progress. Alternatively, we could create a separate PR later when the retrospective process is more concrete and add details then.

**Comment by TheFoxAtWork** [comment]
@PushkarJ i think when the retrospective is more concrete we can do a separate PR to do the update.  thoughts?

**Pull request merged**